src/routes/admin/_admin.transactions.tsx

```diff
@@ -62,6 +62,8 @@ export function Transactions() {
   const [debouncedBlockHeight] = useDebounce(blockHeight, 200);
   const { order_by_field, sort_direction } = useSearch({ from: '/admin/_admin/transactions' });
 
+  const showRecordTransaction = false;
```

Comment:
Please add jsdoc, that we're hiding record transaction button and dialog, until spv-wallet functionality for recording transactions would fulfil users needs and expectations.

Reply:
Sure.